IDEAS/Buildings/Components/Shading/BuildingShade.mo

```diff
@@ -24,37 +32,60 @@ model BuildingShade
   // Computation assumes that window base is at ground level.
   // Viewing angle computed from center of glazing.
 protected
+  final parameter Boolean haveOverhang = LOv > Modelica.Constants.small 
+    "To avoid divisions by zero and obsolete computations";
   parameter Modelica.Units.SI.Angle vieAngObj=atan((hWin/2 + dh)/L)
     "Viewing angle of opposite object";
+  parameter Modelica.Units.SI.Angle vieAngOv=if haveOverhang then Modelica.Constants.pi/2 - atan((hWin/2 + dhOv)/LOv) else 0
+    "Viewing angle of overhang";
```

Comment:
1. Isn't view factor a better word?
2. I would expect `vieAngOve` to be equal to `atan(LOve/(hWin/2+dhOve))`. How did you derive your expression?